src/RedisCache/RedisCache/Commands/NewAzureRedisCache.cs

```diff
@@ -83,6 +83,12 @@ public class NewAzureRedisCache : RedisCacheCmdletBase
         [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "Redis version. Valid values: 4, 6")]
         public string RedisVersion { get; set; }
 
+        [Parameter(ValueFromPipelineByPropertyName = true, Mandatory = false, HelpMessage = "Specifies the type of identity used for the Azure Cache for Redis. Valid values: \"SystemAssigned\" or \"UserAssigned\" or \"SystemAssignedUserAssigned\" or \"None\" ")]
+        public string IdentityType { get; set; }
```

Comment:
Please provide a argument completer for IdentityType

Reply:
added